config/routes.rb

```diff
@@ -5,15 +5,23 @@
   # You can have the root of your site routed with "root"
    root 'tasks#index'
 
-   get '/tasks/:id' =>'tasks#show'
-   get '/new' =>'tasks#new'
-   post '/tasks' => 'tasks#create'
-   get '/modify/:id' => 'tasks#modify'
-   patch '/modify/:id' => 'tasks#updated'
+   get '/tasks/:id'           =>  'tasks#show'
```

Comment:
To follow the RESTful route standard you'd want to use the controller name `/tasks/` at the start of the routes so you'd want:
/tasks/new
/tasks/:id/modify
/tasks/:id/delete